components/kyma-environment-broker/internal/process/deprovision_operation.go

```diff
@@ -72,6 +72,7 @@ func (om *DeprovisionOperationManager) RetryOperationOnce(operation internal.Dep
 func (om *DeprovisionOperationManager) RetryOperation(operation internal.DeprovisioningOperation, errorMessage string, retryInterval time.Duration, maxTime time.Duration, log logrus.FieldLogger) (internal.DeprovisioningOperation, time.Duration, error) {
 	since := time.Since(operation.UpdatedAt)
 
+	log.Infof("Retry Operation was triggered with message: %s", errorMessage)
 	log.Infof("Retrying for %s in %s steps, error: %s", maxTime.String(), retryInterval.String(), errorMessage)
```

Comment:
the existing log statement is already logging errorMessage

Reply:
@piotrmiskiewicz it doesn't mention the error message till the operation is failed. once failed then it logs. it would be nice to see the reason behind retry as well, right?

Reply:
I agree with the idea of this PR. You are absolutely right! In the line 76 we are already logging the errorMessage. This is the reason you don't need to add line 75 or I'm missing something?

Reply:
@piotrmiskiewicz when i was debugging, i saw there were some operations which calls retry for like 30 mins, now this RetryOperation will not log the error message till 30 mins, I thought it would be a good idea to have an intermediate log to prompt user that why this RetryOperation was triggered on the first place